### DISCUSSION TITLE
Is there a way to filter some elements on the save method (in order to not save them)

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#5533] [General]
I would like to add an attribute 'skipOnSave' or something similar, and when the save function executes, go through all of them and remove those items (and all of its children) from the data that is sent to the server.

How should I approach this?

**Comment by artf** [comment]
You can extend the [toJSON](https://github.com/GrapesJS/grapesjs/blob/0a651c3eddaddf8e158db1e65f9b14969385ccb9/src/dom_components/model/Component.ts#L1657) method (used for component serialization) and use there your own logic.
